---
title: "EES 590 Reflections: PCR & Primer Design"
description: Week 6 Reflections
author: Samuel Tan
---

While already familiar with PCR theory, and to a lesser extent, primer design, it was nice to be able to pick up some tips on the more practical aspects of the latter. Admittedly, I haven't got to the stage where I have to design primers for my PhD project, but listening to the challenges my peers have been facing in their work thus far, especially with regards to cross reactivity (e.g. Alex Ascher's lobster blockers blocking the Calanus finmarchichus in their diet, cross reactivity in Dara and Shane's fish assays with other fish species).

While I am aware that genomic databases can be inaccurate, especially for more obscure taxa, it had not occurred to me as to the extent which this could affect the performance of primers ex-silico. Just because a primer set works well in-silico may not mean it will work well in the real world, especially if the sequences used to validate its performance are dodgy to begin with! If there are problems with low detection and/or cross reactivity, it is probably better to just barcode your taxa of interest de novo. Existing sequences on databases out there may be mis-assigned, or perhaps intra-specific diversity for the gene in question is much greater than anticipated....

Another important takeaway to me is; given the relatively low cost of Sanger sequencing these days, it does indeed make sense to sequence at least a subset of your suspicious, unexpected amplicons, to figure out what they actually are (e.g. actual off-target taxa , pseudogenes, etc.). 

The Covid PCR paper also brought up interesting discussion points about primer design for something as fast-mutating as viruses. Interesting strategies to deal with this problem include the ue of multiple primer sets for redundancy, regular optimisation, and primer design to target regions with lower mutation rates. 

So far, my usage of PCR for my past projects has been...surprisingly straightforward,  ddPCR aside (which I just can't seem to get to work with my microbial 16S extracts), and I can't say I'm looking forward to the challenges of primer design, much less things like multiplex PCR, which I may actually need to do given that I'm considering using 16S and 18S data from index sites samples for my future project. 