---
title: Metabarcoding and next gen sequencing
description: Sarahs reflection metabarcoding
author: Sarah Hosler
---

This weeks lesson was very helpful for me because it can be applied to my research as well. Eventually I would like to use metabarcoding to identify and sequence the cow gut microbiome in my samples. The paper we discussed in class along with the discussion was really interesting because many people were surprised by the results of the 12S working best for fish and the COI not working as well. I agree that it can be hard to determine which officially works best because there are not the same sized and consistent databases across primer regions. It was also a helpful lesson for our class project because we are going to be using MiFish primers on our snow track samples (if it snows again). I think the bioinformatics part of metabarcoding needs to be worked on the most. In undergrad I did computational research on protein active sites so I have experience using a lot of the sequence databases we have discussed and have done a lot of sequence alignments so when we talk about similar things that I did with proteins it is fun to think back on my undergrad research. Also because of this I ma excited for out bioinformatics lesson next week although itâ€™s a huge field and can be super confusing and exhausting so I would never want to do that as a full time career (sorry Andy and anyone else who does mostly bioinformatics) but that side of research is super interesting and I think everyone should have to do some type of bioinformatics at some point.