---
title: Reflection from primer design module
description: Dara's reflection from discussion of general primers and metabarcoding 
author: dyiu
---

The discussion on Collins et al. 2019 this week was surprisingly relevant to a lot of topics I have been thinking about lately. The authors went through some similar steps that we did when looking for the best gene to use as a fish metabarcode for our eDNA work. We compiled a list of taxa we would likely find in the coastal Gulf of Maine, and used those to compile and compare reference libraries. We came to the same conclusion as this study did here, which was that 12S (specifically the MiFish) is easily the best. I thought it was interesting that after their initial analysis, they felt the need to test three COI primers, instead of looking into other potentially good but less barcoded genes like 16S.  Especially for fish, COI seems more difficult to interpret. Also, what is tRNA-Val?
I thought this paper discussion also nicely complemented some other eDNA discussions from over the past couple of weeks as well. 
